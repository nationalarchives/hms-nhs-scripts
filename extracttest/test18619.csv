data.text

;1
1;
1;1
1; 1
1;   1
 1;  1
  1  ;  1
0
0*0
a*a
0.0
0.0;
;0.0
;
a
0:0
"a,a"
0;a
a;0
0;0
.08;
0.0;0.0
1.0;0.0
0.0;1.0
0.08;1
0.16;1
0.24;1
0.32;1
0.4;1
0.40;1
0.48;1
0.56;1
0.64;1
0.72;1
0.8;1
0.80;1
0.88;1
0.96;1
0.080;1
0.160;1
0.240;1
0.320;1
0.400;1
0.480;1
0.560;1
0.640;1
0.720;1
0.800;1
0.880;1
0.960;1
1;0.08
1;0.16
1;0.24
1;0.32
1;0.4
1;0.40
1;0.48
1;0.56
1;0.64
1;0.72
1;0.8
1;0.80
1;0.88
1;0.96
1;0.080
1;0.160
1;0.240
1;0.320
1;0.400
1;0.480
1;0.560
1;0.640
1;0.720
1;0.800
1;0.880
1;0.960
0.01;
0.04;
0.041;
0.081;
0.12;
0.121;
0.161;
0.20;
0.21;
0.241;
0.28;
0.281;
0.321;
0.36;
0.361;
0.401;
0.44;
0.441;
0.481;
0.52;
0.521;
0.561;
0.60;
0.61;
0.641;
0.68;
0.681;
0.721;
0.76;
0.761;
0.801;
0.84;
0.841;
0.881;
0.92;
0.921;
0.961;
0.99;
O;O
O;1
1;O
o;o
o;1
1;o
 ; 
 ;1
1; 
0;01.08
; 2.16
1; .24
2;03.32
3;04.4
4;05.40
5;06.48
6;07.56
7;08.64
8;09.72
9;10.8
10;11.80
11;12.88
12;13.96
13;14.080
14;15.160
0.25;0.5
0.75;0.33
0.66;0.67
