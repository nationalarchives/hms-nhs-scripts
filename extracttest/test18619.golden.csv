data.text
""
00; 01
01; 00
01; 01
01; 01
01; 01
01; 01
01; 01
0
0*0
a*a
0.0
00; 00
00; 00
00; 00
a
00; 00
a; a
00; a
a; 00
00; 00
00.08; 00
00; 00
01; 00
00; 01
00.08; 01
00.16; 01
00.24; 01
00.32; 01
00.40; 01
00.40; 01
00.48; 01
00.56; 01
00.64; 01
00.72; 01
00.80; 01
00.80; 01
00.88; 01
00.96; 01
00.08; 01
00.16; 01
00.24; 01
00.32; 01
00.40; 01
00.48; 01
00.56; 01
00.64; 01
00.72; 01
00.80; 01
00.88; 01
00.96; 01
01; 00.08
01; 00.16
01; 00.24
01; 00.32
01; 00.40
01; 00.40
01; 00.48
01; 00.56
01; 00.64
01; 00.72
01; 00.80
01; 00.80
01; 00.88
01; 00.96
01; 00.08
01; 00.16
01; 00.24
01; 00.32
01; 00.40
01; 00.48
01; 00.56
01; 00.64
01; 00.72
01; 00.80
01; 00.88
01; 00.96
00; 00
00.08; 00
00.08; 00
00.08; 00
00.16; 00
00.16; 00
00.16; 00
00.24; 00
00.24; 00
00.24; 00
00.32; 00
00.32; 00
00.32; 00
00.40; 00
00.40; 00
00.40; 00
00.48; 00
00.48; 00
00.48; 00
00.56; 00
00.56; 00
00.56; 00
00.64; 00
00.64; 00
00.64; 00
00.72; 00
00.72; 00
00.72; 00
00.80; 00
00.80; 00
00.80; 00
00.88; 00
00.88; 00
00.88; 00
00.96; 00
00.96; 00
00.96; 00
00.96; 00
00; 00
00; 01
01; 00
00; 00
00; 01
01; 00
00; 00
00; 01
01; 00
00; 01.08
00; 02.16
01; 00.24
02; 03.32
03; 04.40
04; 05.40
05; 06.48
06; 07.56
07; 08.64
08; 09.72
09; 10.80
10; 11.80
11; 12.88
12; 13.96
13; 14.08
14; 15.16
00.24; 00.48
00.72; 00.32
00.64; 00.64
